examples/vmi-sriov.yaml

```diff
@@ -30,16 +38,11 @@ spec:
   - name: default
     pod: {}
   - multus:
-      networkName: sriov/sriov-network
+      networkName: sriov-network
     name: sriov-net
   terminationGracePeriodSeconds: 0
   volumes:
   - containerDisk:
-      image: registry:5000/kubevirt/fedora-cloud-container-disk-demo:devel
+      image: registry:5000/kubevirt/fedora-sriov-lane-container-disk:devel
```

Comment:
Wondering what is the advantage here; what does this image have that the other one doesn't ? Why should a user care about this image ? 

The answers to the above should be present in the commit msg.

Reply:
This is image include's all drivers for SRIOV that supported by the `sriov-network-operator`, so one will not have to worry about that.
You can find more info about this image description at https://github.com/kubevirt/kubevirt/blob/master/containerimages/container-disk-images.md#kubevirt-container-disk-images

Reply:
Don't tell it to me, tell it to everyone that uses this, via a neat commit message :) 

Keeping it as a separate commit could also be helpful (since it isolates the motivation for that change really well). Up to you.

The link is helpful, please include it.